README.md

```diff
@@ -1009,7 +1010,7 @@ Table of Contents
 
 
 ## Screenshot APIs
-
+  * [Abstract Screenshot API](https://www.abstractapi.com/website-screenshot-api) — Screenshot API handling resizing, multiple image formats, css injections and javascript rendering.
```

Comment:
Thank you, I'd prefer we list it once only

Reply:
Sure!
Do you expect me to remove and resubmit a PR?